docs/content/docs/advanced/offline-browsing.md

```diff
@@ -74,7 +74,30 @@ Congratulation. Your website is now a offline-first application.
 
 ## FAQ
 
+### How to enable on demand cache for files that weren't cache on initial load ?
+
+This is the default behavior of the builtin Service Worker logic.
+It will cache everything under the same origin on demand.
+
+### Can you provide other useful globby patterns ?
+
+Of course. Here is some useful pattern that covers 90% cases of Phenomic's user.
+(along with the default pattern)
+
+- `["index.html", "**.css", "**.js"]`: Only cache css, js files and the main `index.html`.
```

Comment:
Hum. On the first load, SW is not installed, css and js will not be cached on demand. User must refresh to cache those assets on demand. Is this ok ? Phenomic is so good that I never refresh my blog page :dancer: